base/src/main/proto/spine/options.proto

```diff
@@ -836,3 +844,42 @@ message IsOption {
     //
     string java_type = 2;
 }
+
+// Defines the way to compare two messages of the same type to one another.
+//
+// Comparisons can be used to sort values.
+//
+// See the `(compare_by)` option.
+//
+message CompareOption {
```

Comment:
As discussed, let's call this `CompareByOption`, to leave `Compare...` unengaged for future use.